src/renderer/components/hotbar/hotbar-selector.tsx

```diff
@@ -40,14 +40,16 @@ export function HotbarSelector({ hotbar }: Props) {
     <div className="HotbarSelector flex align-center">
       <Icon material="play_arrow" className="previous box" onClick={() => store.switchToPrevious()} />
       <div className="box grow flex align-center">
-        <MaterialTooltip arrow title={hotbar.name}>
-          <Badge
-            id="hotbarIndex"
-            small
-            label={hotbarDisplayIndex(store.activeHotbarId)}
-            onClick={() => CommandOverlay.open(<HotbarSwitchCommand />)}
-          />
-        </MaterialTooltip>
+        <Badge
```

Comment:
Why are we going back to our in-tree version instead? Wouldn't it be better to get rid of that and move to `MaterialTooltip` instead?

Reply:
Well, the issue was marked to `5.0.0` and revering changes with some restyling became fastest way to solve thing. Going to `MaterialTooltip` way need to refactor `withTooltip` HOC, which is slower and more complex. Probably we should make another attempt on `MaterialTooltip` later.

Reply:
Yes it was marked as `5.0.0` initially.

I guess I thought that the issue was about moving directly to `MaterialTooltip` since we have had over a week to do so. WDYT @leenamba ?

Reply:
This PR fixes the described issue, so I suggest to merge it for 5.0.1